packages/blanket/src/styles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
```

Comment:
It is not a style file, it is a component file. It is better to use the name like `styled` or `components`. In general. we use `styled`

Reply:
Hmm, i am not sure about it, but for now i don't have arguments.